.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang 24.2
```

Comment:
Can we do 25? You can look up the latest version

Reply:
Yeah, 25.0.2 is the latest version so just set this to 25.0
https://www.erlang.org/patches/otp-25.0.2

Reply:
Done

Reply:
Actually, I think that it is better to stick to Erlang 24:
"Elixir is precompiled for OTP 22 which is not fully compatible with OTP 25. You need to either use OTP 24 or compile Elixir from source."

Reply:
:+1: We can revisit at a later time